invenio_communities/communities/services/service.py

```diff
@@ -72,6 +72,14 @@ def rename(self, id_, identity, data, revision_id=None):
         if 'id' not in data:
             raise ValidationError('Missing data for required field.', 'id')
 
+        data, errors = self.schema.load(
+            data,
+            context={"identity": identity},
+            raise_errors=raise_errors,     # if False, flow is continued with
```

Comment:
Let's remove the `raise_errors` as we will always use the default which is True...